modules/post/windows/gather/enum_snmp.rb

```diff
@@ -3,142 +3,192 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
-
 class MetasploitModule < Msf::Post
   include Msf::Post::Windows::Registry
   include Msf::Auxiliary::Report
 
-  def initialize(info={})
-    super( update_info( info,
-        'Name'          => 'Windows Gather SNMP Settings Enumeration (Registry)',
-        'Description'   => %q{ This module will enumerate the SNMP service configuration },
-        'License'       => MSF_LICENSE,
-        'Author'        => [ 'Carlos Perez <carlos_perez[at]darkoperator.com>', 'Tebo <tebo[at]attackresearch.com>'],
-        'Platform'      => [ 'win' ],
-        'SessionTypes'  => [ 'meterpreter' ]
-      ))
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Gather SNMP Settings',
+        'Description' => %q{ This module will enumerate the SNMP service configuration. },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Carlos Perez <carlos_perez[at]darkoperator.com>',
+          'Tebo <tebo[at]attackresearch.com>'
+        ],
+        'References' => [
+          ['MSB', 'MS00-096'],
+          ['URL', 'https://docs.microsoft.com/en-us/security-updates/securitybulletins/2000/ms00-096'],
+        ],
+        'Platform' => [ 'win' ],
+        'SessionTypes' => %w[shell powershell meterpreter],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [],
+          'SideEffects' => []
+        },
+        'Compat' => {
+          'Meterpreter' => {
+            'Commands' => %w[
+              stdapi_registry_open_key
+              stdapi_registry_check_key_exists
+            ]
+          }
+        }
```

Comment:
This can all be left out because it's provided by `Msf::Post::Windows::Registry`.

Reply:
Removed.